Source/Extensions/Blazorise.DataGrid/DataGrid.razor.cs

```diff
@@ -162,7 +162,7 @@ internal void Hook( DataGridColumn<TItem> column )
         /// Links the child column with this datagrid.
         /// </summary>
         /// <param name="aggregate">Aggregate column to link with this datagrid.</param>
-        internal void Hook( DataGridAggregate<TItem> aggregate )
+        public void AddAggregate( DataGridAggregate<TItem> aggregate )

```

Comment:
Same as the columns above, adding these type of Razor components programatically.

Reply:
> I've done this on Virtualize PR, there's a few places where in my opinion, would benefit from refactoring the code blocks into a simple more concise Reload() / SetDirty() Since I'm not sure we'll be able to get Virtualize into 0.9.4, might be good to bring those changes over here.
> [David-Moreira/Blazorise@8cc752a](https://github.com/David-Moreira/Blazorise/commit/8cc752a12b3589b7d3addc7bf63b0997db5b3253)
> [David-Moreira/Blazorise@f6d1640](https://github.com/David-Moreira/Blazorise/commit/f6d16403d416b5237d688cb2e05f0dc17d475cd0)
> 
> ![image](https://user-images.githubusercontent.com/22283161/123527482-e13b3a80-d6d7-11eb-8a2f-294a8a52dab8.png)

I'm not sure what I need to do for HandleReadData. Can you explain? It is better to be in InvokeAsync because otherwise, users could get deadlock in some rare situations. I know we had it on one of the client projects. For every action that doesn't come from the Blazor event, we must wrap it in InvokeAsync.

Reply:
Nah, just saying the code block is easily replaced by a call to `Reload()` in several places.

Also there are several` dirtyFilter = dirtyView = true;` across the code, those would also benefit from being inside a Method `SetDirty()`

Reply:
Also not sure why you quoted that comment in here. >.<

Reply:
Haha quote was done by the GitHub somehow